app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, NotWearingMaskError
```

Comment:
You should import `VaccineError` instead of `NotVaccinatedError` to handle both `NotVaccinatedError` and `OutdatedVaccineError` in a single `except` clause, as specified in the task requirements.